tarantool.lua

```diff
@@ -241,7 +241,7 @@ do
         return response.data
       end
     end,
-    update = function(self, index, key, oplist)
+    update = function(self, space, index, key, oplist)
```

Comment:
The change looks correct, but shouldn't we update the `tarantool.moon` file and regererate `tarantool.lua` to keep them in sync?

Reply:
Updated `tarantool.moon` and regenerated `tarantool.lua`.